app/controllers/api/base_controller/renderer.rb

```diff
@@ -165,6 +165,18 @@ def collection_search(is_subcollection, type, klass)
         Rbac.filtered(res, options)
       end
 
+      def virtual_attribute_search(resource, attribute)
+        if resource.class < ApplicationRecord
+          if resource.class.try(:plural?, attribute)
+            Rbac.filtered(resource.public_send(attribute))
+          else
+            Rbac.filtered_object(resource).try(:public_send, attribute)
```

Comment:
is the Rbac.filtered_object(resource) on line 173 needed ?  We already did the RBAC on the resource, isn't resource.public_send(attribute) sufficient ?  Thanks.